backend/src/LeanCode.AppRating/EmailViewModels/LowRateSubmittedEmail.cs

```diff
@@ -0,0 +1,6 @@
+namespace LeanCode.AppRating.EmailViewModels;
+
+public class LowRateSubmittedEmail
+{
+    public string? UserId { get; set; }
```

Comment:
Let's include the rating, and comments, directly in the model. Would save a lot of hassle for end users of this lib.